Userland/Libraries/LibWeb/MimeSniff/Resource.cpp

```diff
@@ -0,0 +1,629 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Resource.h"
```

Comment:
The include style is typically the full path:
```suggestion
#include <LibWeb/MimeSniff/Resource.h>
```